src/constant.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 1989-94 James E. Wilson, Robert A. Koeneke
+// Copyright (c) = 1989-94 James E. Wilson, Robert A. Koeneke
```

Comment:
It looks like your search/replace has also changed a bunch of comments by adding an `=` sign, here and throughout the file. Line #41, 42, 50, etc., etc.